rocAL/rocAL_pybind/amd/rocal/plugin/pytorch.py

```diff
@@ -35,26 +35,27 @@ def __init__(self, pipeline, tensor_layout = types.NCHW, reverse_channels = Fals
         self.device = device
         self.device_id = device_id
         self.batch_size = self.loader._batch_size
-        self.out = self.dimensions = self.torch_dtype = None
+        self.output = self.dimensions = self.torch_dtype = None
         self.len = b.getRemainingImages(self.loader._handle)
+        self.display = display
 
     def next(self):
         return self.__next__()
 
     def __next__(self):
-        if(b.isEmpty(self.loader._handle)):
+        if self.loader.rocal_run() != 0:
             raise StopIteration
         else:
-            self.output_tensor_list = self.loader.getOutputTensors()
+            self.output_tensor_list = self.loader.get_output_tensors()
 
-        if self.out is None:
+        if self.output is None:
             self.dimensions = self.output_tensor_list[0].dimensions()
             torch_gpu_device = torch.device('cuda', self.device_id)
             self.torch_dtype = self.output_tensor_list[0].dtype()
-            self.out = torch.empty((self.dimensions[0], self.dimensions[1], self.dimensions[2], self.dimensions[3],), dtype = getattr(torch, self.torch_dtype), device = torch_gpu_device)
+            self.output = torch.empty((self.dimensions[0], self.dimensions[1], self.dimensions[2], self.dimensions[3],), dtype = getattr(torch, self.torch_dtype), device = torch_gpu_device)
             self.labels_tensor = torch.empty(self.dimensions[0], dtype = torch.int32, device = torch_gpu_device)
 
-        self.output_tensor_list[0].copy_data(ctypes.c_void_p(self.out.data_ptr()))
+        self.output_tensor_list[0].copy_data(ctypes.c_void_p(self.output.data_ptr()))
         if((self.loader._name == "Caffe2ReaderDetection") or (self.loader._name == "CaffeReaderDetection")):
             self.lis = []  # Empty list for bboxes
```

Comment:
Please change name lis to something meaningful.

Reply:
Decided to remove this file from the pybind PR and include it in readers PR